src/f4enix/input/MCNPinput.py

```
@@ -44,11 +44,12 @@
     PAT_NP,
     UNION_INTERSECT_SYMBOLS,
 )
-from f4enix.input.auxiliary import debug_file_unicode
+from f4enix.core.auxiliary import debug_file_unicode
 from f4enix.input.d1suned import IrradiationFile, Reaction, ReactionFile
-from f4enix.input.irradiation import METASTABLE_TAG, Nuclide
+from f4enix.core.irradiation import METASTABLE_TAG, Nuclide
 from f4enix.input.libmanager import LibManager
 from f4enix.input.materials import MatCardsList, Material
+from f4enix.core.irradiation import Nuclide, METASTABLE_TAG
```

Ruff (0.14.14)

[error] Redefinition of unused `Nuclide` from line 49: `Nuclide` redefined here

Remove definition: `Nuclide`

(F811)